test/specs/carousel.spec.js

```diff
@@ -518,6 +518,20 @@ describe('Carousel', function() {
       teardown();
     });
 
+    it('should call onResize prop on onResize handler', () => {
+      component = ReactDOM.render(
+        React.createElement(carousel, null,
+          React.createElement('p', null, 'Slide 1'),
+          React.createElement('p', null, 'Slide 2'),
+          React.createElement('p', null, 'Slide 3')
+        ),
+        container
+      );
+      const spy = sinon.spy(component, 'onResize');
+      component.onResize();
```

Comment:
This test isn't testing anything. You've _spied_ `component.onResize`, then **directly called it**.

You need to spy on `props.onResize` and call `component.onResize` at a minimum.